app/experimenter/experiments/constants/nimbus.py

```diff
@@ -858,13 +965,25 @@ class EmailType(models.TextChoices):
         TARGETING_INFREQUENT_USER_NEED_DEFAULT.slug: (
             TARGETING_INFREQUENT_USER_NEED_DEFAULT
         ),
+        TARGETING_INFREQUENT_USER_NEED_DEFAULT_HAS_PIN.slug: (
+            TARGETING_INFREQUENT_USER_NEED_DEFAULT_HAS_PIN
+        ),
+        TARGETING_INFREQUENT_USER_HAS_DEFAULT_NEED_PIN.slug: (
+            TARGETING_INFREQUENT_USER_HAS_DEFAULT_NEED_PIN
+        ),
```

Comment:
These two (and the other four that are already here) need to be added into `class TargetingConfig(models.TextChoices):`